src/components/summary/_macro.njk

```diff
@@ -20,100 +20,110 @@
-                                                        {% if action.attributes is defined and action.attributes %}{% for attribute, value in (action.attributes.items() if action.attributes is mapping and action.attributes.items else action.attributes) %}{{attribute}}="{{value}}" {% endfor %}{% endif %}
-                                                    >{{ action.text }}</a>
-                                                {% endfor %}
-                                            </td>
-                                        {% endif %}
-                                    </tr>
-                                {% endfor %}
-                            </tbody>
-                        {% endfor %}
-                    </table>
+                                                </td>
+                                            {% endif %}
+                                            {% if rowItem.actions is defined and rowItem.actions %}
+                                                <td class="summary__actions">
+                                                    {% for action in (rowItem.actions if rowItem.actions is iterable else rowItem.actions.items()) %}
+                                                        {% if loop.index > 1 %}<span class="summary__spacer"></span>{% endif %}
+                                                        <a
+                                                            href="{{ action.url }}"
+                                                            class="summary__button"
+                                                            {% if action.ariaLabel is defined and action.ariaLabel %} aria-label="{{ action.ariaLabel | striptags() }}"{% endif %}
+                                                            {% if action.attributes is defined and action.attributes %}{% for attribute, value in (action.attributes.items() if action.attributes is mapping and action.attributes.items else action.attributes) %}{{attribute}}="{{value}}" {% endfor %}{% endif %}
+                                                        >{{ action.text }}</a>
+                                                    {% endfor %}
+                                                </td>
+                                            {% endif %}
+                                        </tr>
+                                    {% endfor %}
+                                </tbody>
+                            {% endfor %}
+                        </table>
+                    {% elif group.placeholderText is defined and group.placeholderText %}
+                        {{ group.placeholderText }}
+                    {% endif %}
+
+                    {% if group.summaryLink is defined and group.summaryLink %}
+                        <div class="summary__add-link{% if group.placeholderText is not defined and group.rows is defined and group.rows %} u-bt{% endif %}">
```

Comment:
rather than the negative bottom margin we put on `.summary__items` to avoid the double border, could we just put something like if group.rows > 1 for that border class, so it’s not added if there’s only one row in the summary which has it’s own border?